Azure-Samples/modern-data-warehouse-dataops
# ydaponte/580 582 smart ingest with config file

# Type of PR

- Code changes

## Purpose
- Add a configuration file that should drive the smart ingestion in the Data Lake.
- Modify the existing pipeline to ingest or update data lake based on configuration file.

## Does this introduce a breaking change? No.

## Author pre-publish checklist
- [ ] Made corresponding changes to the documentation (not yet)

## Validation steps
- Deploy the sample as per the instructions
- Run the Pl_NYCTaxi_0_Main pipeline
- Check on the Monitor the pipeline run should succeed

## Issues Closed or Referenced
- Closes #580 
- Closes #582 

- References #578
